plugins/collections/get.js

```diff
@@ -168,12 +51,12 @@ module.exports = () => ({
 
                     return (
                         Promise.all(collectionPipelines)
-                            // Populate pipelines with PR Info and then last builds
-                            .then(pipelines => Promise.all(getPipelinesInfo(pipelines, eventFactory)))
-                            .then(pipelinesWithInfo => {
-                                result.pipelines = pipelinesWithInfo;
+                            // to filter out null
+                            .then(pipelines => pipelines.filter(pipeline => pipeline))
```

Comment:
What does filter do here?

`pipelines.filter(pipeline => pipeline))`

 Is it to filter out nullish values?

Reply:
yes